src/main/java/io/discloader/discloader/core/entity/guild/GuildMember.java

```diff
@@ -85,8 +80,7 @@
 	public final Date joinedAt;
 	
 	public GuildMember(Guild guild, MemberJSON data) {
```

Comment:
Change `public GuildMember(Guild guild, MemberJSON data)` to `public GuildMember(IGuild guild, MemberJSON data)`